modules/BiotypeQC.nf

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env nextflow
+
+/*
+BioQC is a is a R/Bioconductor package to detect tissue heterogeneity in gene expression data
+*/
+
+params.gene_expression = "/home/ausserh/myScratch/Salmon_test.txt"
+params.samplesheet = "/home/ausserh/testMatrix.tsv"
+params.marker = "extdata/exp.tissuemark.affy.roche.symbols.gmt"
+
+
+process 'BioQC' {
+
+  conda 'bioconductor-bioqc r-readr'
+  publishDir "/home/ausserh/BioQC_test", mode: 'copy'
```

Comment:
We'll handle this via the `nextflow.config`:
https://github.com/icbi-lab/nf-deseq2/blob/b61ce89146e6a92346c180fcf71574d5aef145c7/nextflow.config#L6-L9

Could you add a publishDir directive there?